sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Inject, Injectable } from "@nestjs/common";
+import {
+  EducationProgramOffering,
+  EducationProgram,
+  InstitutionLocation,
+} from "../../database/entities";
+import { RecordDataModelService } from "../../database/data.model.service";
+import { Connection } from "typeorm";
+import { CreateEducationProgramOfferingDto } from "src/route-controllers/education-program-offering/models/create-education-program-offering.dto";
```

Comment:
Please remove the "src" from the path and make the import relative.

Reply:
done